tests/framework/population/test_population_view.py

```diff
@@ -245,24 +258,24 @@ def test_get_fail(population_manager):
 
 
 def test_full_population_view__coerce_to_dataframe(
-    population_update,
-    update_index,
-):
+    population_update: pd.Series[Any] | pd.DataFrame,
+    update_index: pd.Index[int],
+) -> None:
     cols = (
         population_update.columns
         if isinstance(population_update, pd.DataFrame)
         else [population_update.name]
     )
 
     coerced_df = PopulationView._coerce_to_dataframe(population_update, COL_NAMES)
-    assert BASE_POPULATION.loc[update_index, cols].equals(coerced_df)
+    assert BASE_POPULATION.loc[update_index, cols].equals(coerced_df)  # type: ignore[index]
```

Comment:
I can't seem to figure out how to make mypy happy w/ `pd.loc`. If anyone has figured this out lmk! There are a few instances where I just ignored `[index]` b/c of this.

Reply:
Thanks @hussain-jafari !